checker/tests/nullness-extra/issue607/Makefile

```diff
@@ -0,0 +1,9 @@
+.PHONY: all
+
+all: clean
+	$(JAVAC) -source 8 -target 8 Issue607SuperClass.java Issue607Interface.java
+	$(JAVAC) -processor org.checkerframework.checker.nullness.NullnessChecker -source 8 -target 8 Issue607.java > Out.txt 2>&1
```

Comment:
Having the `Out.txt` and `Expected.txt` is OK, but it's actually enough to just run the compiler. Removing these files would simplify the test a bit.